testing/tb.go

```diff
@@ -0,0 +1,26 @@
+package testing
+
+var _ TB = (*T)(nil)
+var _ TB = (*F)(nil)
+
+// TB is the interface common to T, B, and F.
+type TB interface {
+	Cleanup(func())
+	Error(args ...any)
+	Errorf(format string, args ...any)
+	Fail()
+	FailNow()
+	Failed() bool
+	Fatal(args ...any)
+	Fatalf(format string, args ...any)
+	Helper()
+	Log(args ...any)
+	Logf(format string, args ...any)
+	Name() string
+	Setenv(key, value string)
+	Skip(args ...any)
+	SkipNow()
+	Skipf(format string, args ...any)
+	Skipped() bool
+	TempDir() string
```

Comment:
This is what I see [as of Go 1.25](https://cs.opensource.google/go/go/+/go1.25.0:src/testing/testing.go;l=881):

```suggestion

	Attr(key, value string)
	Cleanup(func())
	Error(args ...any)
	Errorf(format string, args ...any)
	Fail()
	FailNow()
	Failed() bool
	Fatal(args ...any)
	Fatalf(format string, args ...any)
	Helper()
	Log(args ...any)
	Logf(format string, args ...any)
	Name() string
	Setenv(key, value string)
	Chdir(dir string)
	Skip(args ...any)
	SkipNow()
	Skipf(format string, args ...any)
	Skipped() bool
	TempDir() string
	Context() context.Context
	Output() io.Writer
```

Reply:
Yeah, though you should add these with a go 1.25 build tag.

Reply:
(I am not bothering unless I see some action on this PR - it's been dormant for a year)

Reply:
Sorry for the delay. I am happy to review this, but it will take some testing against OSS-Fuzz projects to make sure we don't break anything. Feel free to update and I will make this a priority this week and next.